content/en/docs/usage/certificate.md

```diff
@@ -167,37 +167,81 @@ If you would prefer the `Secret` to be deleted automatically when the `Certifica
 
 ## Renewal
 
+
 `cert-manager` automatically renews issued certificates. It calculates _when_ to
-renew a certificate based on the X.509 certificate's `duration` as well as the
-`renewBefore` value. The `renewBefore` value specifies _how long_ before expiry
-a certificate should be renewed.
+renew a certificate based Certificate's [`duration`][certspec] and
```

Comment:
Suggestion: '`Certificate's` duration' -> 'certificate's duration' as it was in the original text. We use the actual duration to calculate the renewal time. The `spec.duration` is not used when the renewal time gets calculated.

Reply:
By certificate (without the capital C), do you mean X.509 certificate? In that case, I think it would be good to prepend "X.509" to the word "certificate" in order to disambiguate certificate vs. Certificate.

> The `spec.duration` is not used when the renewal time gets calculated.

Aah right, I just looked at jetstack/cert-manager@master/pkg/controller/certificates/util.go#L272-L291 and I now understand what you mean 😅

Reply:
Yeah I think the `spec.duration` is only used when issuing a cert